internal/config/application.go

```diff
@@ -126,6 +127,17 @@ func (cfg *Application) parseConfigValues() error {
 	return nil
 }
 
+func (cfg *Application) parseFile() error {
+	if cfg.File != "" {
+		expandedPath, err := homedir.Expand(cfg.File)
```

Comment:
If this returns an error, would it still be better to just try the user provided string?

Reply:
fair question --possibly, since I don't have a strong understanding of the failure modes of the `homedir` package. Let me do a little bit of reading and get back to you.

Reply:
:shipit: